frontend/components/map.module.css

```diff
@@ -0,0 +1,21 @@
+.mappage {
+  height: 100vh;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
```

Comment:
lets use css-grid instead of flex much easier to understand

Reply:
(I'm on board -- just going to drag my toes a bit on the layout items until we decide which tool we'll use and are building the real deal map)

Reply:
We talked about this in my PR & it seemed the consensus was to use css-grid for multi-dimensional layouts and flexbox for single-dimensional/content-based layouts 
(aligns with the intended differences between them - here's an article I found helpful! https://css-tricks.com/css-grid-replace-flexbox/)